vNext/extensions/applicationinsights-react-native/package.json

```diff
@@ -0,0 +1,40 @@
+{
+    "name": "@microsoft/applicationinsights-react-native",
+    "version": "0.0.1",
+    "description": "Microsoft Application Insights React Native Plugin",
+    "main": "dist-esm/index.js",
+    "types": "dist-esm/index.d.ts",
+    "sideEffects": false,
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/Microsoft/ApplicationInsights-JS/tree/master/extensions/applicationinsights-properties-js"
+    },
+    "scripts": {
+        "build": "npm run build:esm && npm run build:browser",
+        "build:esm": "grunt reactnative",
+        "build:browser": "echo \"No browser version\"",
+        "test": "grunt reactnativetests"
+    },
+    "devDependencies": {
+        "typescript": "^3.3.3333",
+        "grunt": "1.0.1",
+        "grunt-contrib-qunit": "^3.1.0",
+        "grunt-contrib-uglify": "3.1.0",
+        "grunt-ts": "^6.0.0-beta.15",
+        "rollup-plugin-node-resolve": "^3.4.0",
+        "rollup-plugin-replace": "^2.1.0",
+        "rollup-plugin-uglify": "^6.0.0",
+        "rollup": "^0.66.0",
+        "react-native-device-info": "^0.26.4"
+    },
+    "dependencies": {
+        "@microsoft/applicationinsights-core-js": "1.0.0-beta.5",
```

Comment:
2.0.0-beta is available now, let use that one

Reply:
updated